### PR TITLE
[Docs] Add section about EEPROM on ARM

### PR DESCRIPTION
**Pull request by yanfali**
Document ARM EEPROM issues in FAQ

## Description

Provide guidance for users of ARM based QMK boards

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/Layout/Userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/faq_build.md [hunk 1]
```diff
@@ -132,3 +132,11 @@ brew uninstall --force avr-gcc
...
 brew link --force avr-gcc@7
 ```
+
+### I just flashed my keyboard and it does nothing/keypresses don't register - it's also ARM (rev6 planck, clueboard 60, hs60v2, etc...) (Feb 2019)
+Due to how EEPROM works on ARM based chips, saved settings may no longer be valid.  This affects the default layers, and *may*, under certain circumstances we are still figuring out, make the keyboard unusable.  Resetting the EEPROM will correct this.
+
+[Planck rev6 reset EEPROM](https://cdn.discordapp.com/attachments/473506116718952450/539284620861243409/planck_rev6_default.bin) can be used to force an eeprom reset. After flashing this image, flash your normal firmware again which should restore your keyboard to _normal_ working order.
+[Preonic rev3 reset EEPROM](https://cdn.discordapp.com/attachments/473506116718952450/537849497313738762/preonic_rev3_default.bin)
+
+If bootmagic is enabled in any form, you should be able to do this too (see docs and keyboard info for specifics).
```
Comment (drashna) [inline docs/faq_build.md]:
```suggestion
If bootmagic is enabled in any form, you should be able to do this too (see [Bootmagic docs](feature_bootmagic.md) and keyboard info for specifics on how to do this).
```

**Comment by drashna** [comment]
Also, it should be noted that the compiled images above may need to be updated in the future.  

And that i'm the one that compiled them, and all I did was add `eeconfig_init()` to `matrix_init_user()`, so you will want to reflash "your" firmware, after flashing these ones

**Comment by yiancar** [comment]
I also think instead of linking .bin for specific folders we could generate a general bin that would just do an erase eeprom.

**Comment by drashna** [comment]
@yiancar the bin's are more of a stopgap for the boards that are definitely having this issue.

If it's possible, it would be nice to have a dfu-util command to reset the eeprom. If that's a bin, that's fine too.

**Comment by yiancar** [comment]
Just flashing a .bin full of 0xff would erase the whole of the flash (and therefor the "eeprom")

**Comment by drashna** [comment]
Like completely 0xFF, or with addresses, etc? 

Aka, do you have such a hex?

**Pull request merged**